### ISSUE TITLE
Desktop and mobile profile pages for Coreels website

### ISSUE THREAD

**Issue by JET-Pro** [zuri-training/Team-118-Col-Films#33]
Desktop design links
Following- https://www.figma.com/proto/EbRhvvKMHfpo7YBvgsgbHG/Coreel?page-id=139%3A292&node-id=1011%3A9873&starting-point-node-id=971%3A7146&show-proto-sidebar=1&scaling=scale-down
My Playlists- https://www.figma.com/proto/EbRhvvKMHfpo7YBvgsgbHG/Coreel?page-id=139%3A292&node-id=1109%3A16986&starting-point-node-id=971%3A7146&show-proto-sidebar=1&scaling=scale-down
Watch Later-https://www.figma.com/proto/EbRhvvKMHfpo7YBvgsgbHG/Coreel?page-id=139%3A292&node-id=1109%3A17678&starting-point-node-id=971%3A7146&show-proto-sidebar=1&scaling=scale-down
Downloads-https://www.figma.com/proto/EbRhvvKMHfpo7YBvgsgbHG/Coreel?page-id=139%3A292&node-id=1109%3A18046&starting-point-node-id=971%3A7146&show-proto-sidebar=1&scaling=scale-down
Profile Settings- https://www.figma.com/proto/EbRhvvKMHfpo7YBvgsgbHG/Coreel?page-id=139%3A292&node-id=1109%3A18327&starting-point-node-id=971%3A7146&show-proto-sidebar=1&scaling=scale-down
Sign out- https://www.figma.com/proto/EbRhvvKMHfpo7YBvgsgbHG/Coreel?page-id=139%3A292&node-id=1109%3A18662&starting-point-node-id=971%3A7146&show-proto-sidebar=1&scaling=scale-down

**Comment by JET-Pro** [comment]
Mobile design links
Following- https://www.figma.com/proto/EbRhvvKMHfpo7YBvgsgbHG/Coreel?page-id=139%3A292&node-id=1037%3A9914&starting-point-node-id=1125%3A21825&show-proto-sidebar=1&scaling=scale-down
My Playlists- https://www.figma.com/proto/EbRhvvKMHfpo7YBvgsgbHG/Coreel?page-id=139%3A292&node-id=1095%3A10053&starting-point-node-id=1125%3A21825&show-proto-sidebar=1&scaling=scale-down
Watch Later- https://www.figma.com/proto/EbRhvvKMHfpo7YBvgsgbHG/Coreel?page-id=139%3A292&node-id=1095%3A10255&starting-point-node-id=1125%3A21825&show-proto-sidebar=1&scaling=scale-down
Downloads- https://www.figma.com/proto/EbRhvvKMHfpo7YBvgsgbHG/Coreel?page-id=139%3A292&node-id=1095%3A10803&starting-point-node-id=1125%3A21825&show-proto-sidebar=1&scaling=scale-down
Profile Settings- https://www.figma.com/proto/EbRhvvKMHfpo7YBvgsgbHG/Coreel?page-id=139%3A292&node-id=1095%3A11327&starting-point-node-id=1125%3A21825&show-proto-sidebar=1&scaling=scale-down
Sign out- https://www.figma.com/proto/EbRhvvKMHfpo7YBvgsgbHG/Coreel?page-id=139%3A292&node-id=1095%3A11595&starting-point-node-id=1125%3A21825&show-proto-sidebar=1&scaling=scale-down

**Comment by Enigmas01** [comment]
What's this? And how did it got to be here?
@JET-Pro

**Comment by JET-Pro** [comment]
I have checked it and it's in team 118's. I created it as an issue in my team's project

**Comment by Enigmas01** [comment]
That's true. I just checked it now. May be it's been moved. Last night it was on Team_91 repo. Anyways, well done!

**Comment by Enigmas01** [comment]
This issue is still on our repo — Team_91.
Check the screenshot attached below. I don't know if it's a bug or something. But please check it from your end, I don't want to delete it from my end so it won't affect you or someone else. Thank you.


![Image](https://user-images.githubusercontent.com/105251965/183594613-41371ab3-e07b-4554-8e51-aaa9165cae6a.jpg)
